docs/powerpoint/insert-slides-into-presentation.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Insert and delete slides in a PowerPoint presentation
+description: 'Learn how to insert slides from one presentation into another and how to delete slides.'
+ms.date: 12/04/2020
+localization_priority: Normal
+---
+
+# Insert and delete slides in a PowerPoint presentation
+
+A PowerPoint add-in can insert slides from one presentation (in base64 format) into the current presentation by using PowerPoint's application-specific JavaScript library. You can control whether the inserted slides keep the formatting of the source presentation or the formatting of the target presentation. You can also delete slides from the presentation.
+
+There are two major steps to inserting slides from one presentation into another.
```

Comment:
Are there minor steps? Maybe we could remove "major" so it doesn't seem like we're hiding extra work.
```suggestion
There are two steps to inserting slides from one presentation into another.
```